app/assets/javascripts/controllers/ems_common/ems_common_form_controller.js

```diff
@@ -327,6 +327,7 @@ ManageIQ.angular.app.controller('emsCommonFormController', ['$http', '$scope', '
       $scope.emsCommonModel.default_api_port = "8443";
     } else if ($scope.emsCommonModel.emstype === 'vmware_cloud') {
       $scope.emsCommonModel.default_api_port = "443";
+      $scope.emsCommonModel.event_stream_selection = "none";
```

Comment:
I've actually already spent some time on getting this chosen by default but gave up eventually. I was trying to go with `selected` option:
```javascript
options_for_select([...], {selected: ..., disabled: ...})
```
 in this file [here](https://github.com/ManageIQ/manageiq/blob/master/app/views/layouts/angular-bootstrap/_endpoints_angular.html.haml#L86), but that didn't work so I gave up.
Anyway, thanks for helping me out 😄